### PR TITLE
Update foreman_templates to 6.0.0

### PR DESCRIPTION
**Pull request by ares**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.17
* [ ] 1.16
* [ ] 1.15

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

**Comment by mmoll** [comment]
I guess the wrong package got updated here. ;)
Similar to the RPM, the new foreman requirement should get specified in the package dependency also.

**Comment by ares** [comment]
sorry updated, I relied to much on `bump_rpm.sh` :-) I know it prints TODO, now I understand what I should check :-)

**Comment by ares** [comment]
interesting, this is actually output of
```
./scripts/update_package.rb -m 'Update foreman_templates to 6.0.0' -n foreman_templates -v 6.0.0 
```
with
```
./scripts/update_package.rb -m 'Update foreman_templates to 6.0.0' -n ruby-foreman-templates -v 6.0.0
```

it seems to work fine :-)

**Comment by ares** [comment]
Thanks, all should be addressed now

File: plugins/ruby-foreman-templates/debian/gem.list [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # list of gem urls to download via Jenkins, space separated
-GEMS="https://rubygems.org/downloads/foreman_templates-5.0.1.gem"
+GEMS="https://rubygems.org/downloads/foreman_templates-6.0.0.gem"
 GEMS="$GEMS https://rubygems.org/downloads/diffy-3.0.7.gem"
 GEMS="$GEMS https://rubygems.org/downloads/git-1.2.9.1.gem"
```
Comment (mmoll) [inline plugins/ruby-foreman-templates/debian/gem.list]: as you're at it, could you update these two dependency gems, please? :)

**Comment by mmoll** [comment]
merged with minor modifications, díky @ares!

**Pull request merged**